docs/releases/6.0.txt

```diff
@@ -265,6 +265,28 @@ of each library are the first to add or confirm compatibility with Python 3.12:
 * ``sqlparse`` 0.5.0
 * ``tblib`` 3.0.0
 
+Custom ORM expressions should return params in a ``tuple``
+----------------------------------------------------------
+
+Prior to Django 6.0, :doc:`custom lookups </ref/models/lookups>` and
+:ref:`expressions <writing-your-own-query-expressions>` implementing the ``as_sql``
+method (and their underlying parts ``process_lhs`` and ``process_rhs``) were
+permitted to return a sequence of params in either a list or a tuple. To
```

Comment:
Interestingly, django-stubs had this typed as list [only](https://github.com/typeddjango/django-stubs/blob/bda19560285a2b608bcf193f49e11530dc58ff7f/django-stubs/db/models/sql/compiler.pyi#L18).

Reply:
Updated in https://github.com/typeddjango/django-stubs/pull/2673